### ISSUE TITLE
Repeat Monthly and Yearly missing

### ISSUE THREAD

**Issue by davidhedlund** [Etar-Group/Etar-Calendar#177]
In Samsung S Planner I can repeat **Monthly (on day _x_)** and **Yearly  (on day _x_ Month)**. This is not possible in Etar.

It would be very useful to have them.

**Comment by jspricke** [comment]
Create an event -> repetition -> enable (top right) -> click on repeat weekly -> repeat monthly (yearly). **x** is the day you chose for the single event first. Is this what you mean? Otherwise please give an example.

**Comment by davidhedlund** [comment]
@jsharkey You are right, I didn't _see_ it. Can you please add a border around the drop down menu to make it more visible?

**Comment by davidhedlund** [comment]
@jsharkey Thank you for helping me out too. =)

**Comment by smichel17** [comment]
@davidhedlund This is for the dark theme only, correct?

**Comment by davidhedlund** [comment]
@smichel17 No.
